src/components/Reactions/EmojiReactionBubble.js

```diff
@@ -69,7 +69,6 @@ function EmojiReactionBubble(props) {
 
                 props.onPress();
             }}
-            onLongPress={props.onReactionListOpen}
```

Comment:
Why `onLongPress` was removed?

Reply:
When you look at `PressableWithSecondaryInteraction` implementation you can see that `onSecondaryInteraction` is used instead of `onLongPress`, so here it is redundant:

```ts
onLongPress={props.onReactionListOpen}
onSecondaryInteraction={props.onReactionListOpen}
```

Reply:
Right, thanks